wordpress-mobile/WordPress-iOS
# Issue/7860 insert medias alt field

Fixes #7860

To test:

- check that the img tag's value can be loaded and is editable at the image details edit screen (tap on image to edit its details)
- check if after adding an image with an existing alt tag, the value appears in html img tag
- check if after adding an alt value to an image, the value appears in html img tag
- check that altering the alt value of an image. does not affect the alt value of an ImageAttachment
- check if the alt value (when it exists), is the one which is read when Voice over Accessibility feature is active

Needs review: @diegoreymendez @frosty
